lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -238,6 +244,47 @@ def _partition_remote(
 
         return elements
 
+    @staticmethod
+    def _partition_remote(
+        file: BinaryIO,
+        aryn_api_key: str,
+        aryn_partitioner_address=DEFAULT_ARYN_PARTITIONER_ADDRESS,
+        threshold: float = 0.4,
+        use_ocr: bool = False,
+        ocr_images: bool = False,
+        ocr_tables: bool = False,
+        extract_table_structure: bool = False,
+        extract_images: bool = False,
+        selection_size: int = 25,
+    ) -> List[Element]:
+        file.seek(0)
+        pdf_reader = PyPDF2.PdfReader(file)
+        page_count = len(pdf_reader.pages)
+        file.seek(0)
+
+        result = []
+        low = 0
+        high = selection_size - 1
+        while low < page_count:
+            result.extend(
```

Comment:
Did we ever change the result to be a dict, or is it still just a list of elements? Seems here that it's still a list, but I thought we decided otherwise. If we change it, this will probably need to change as well.

Reply:
Yes, we have changed the result to be a dict, and this is accounted for: see [`detr_partitioner:237`](https://github.com/aryn-ai/sycamore/blob/7e2373de6ec1f4f07a00e0ea8d59cb2652ab59fe/lib/sycamore/sycamore/transforms/detr_partitioner.py#L237)